docs/reference/policies/versioning.md

```diff
@@ -3,8 +3,12 @@
 uv uses a custom versioning scheme in which the minor version number is bumped for breaking changes,
 and the patch version number is bumped for bug fixes, enhancements, and other non-breaking changes.
 
-uv does not yet have a stable API; once uv's API is stable (v1.0.0), the versioning scheme will
-adhere to [Semantic Versioning](https://semver.org/).
+uv is widely used and we believe it is production-ready. However, we value the ability to iterate on
+new features quickly and gather changes that _could_ be breaking into clearly marked releases.
```

Comment:
I think we can say something stronger than this. Like: "uv is widely used in production."